server.js

```diff
@@ -21,6 +21,7 @@ app.prepare().then(() => {
   server.use(morgan('tiny'))
 
   server.get('/about', (req, res) => app.render(req, res, '/about'))
+  server.get('/preconfigured-editor', (req, res) => app.render(req, res, '/preconfigured-editor'))
```

Comment:
This is pretty long for a path – maybe shorten it to something like `/configured`?